contrib/debug-rn-examples.md

```diff
@@ -0,0 +1,124 @@
+# How to Debug using React Native Examples
+
+## Steps to link local `realm` and `@realm/react` to examples
+
+### 1. Add example to `workspaces` in `packages.json`
+For example, to add `rn-todo-list` on would add it like so:
+```json
+...
+"workspaces": [
+	"examples/rn-todo-list/frontend/",
+	"packages/realm/bindgen/",
+	"packages/realm/bindgen/vendor/realm-core/",
+	"packages/babel-plugin",
+...
+```
+
+### 2. Update relative paths to `node-modules` in native configurations
+
```

Comment:
I think it could be helpful to mention that the changes shown in this section is for using the root node_modules (just a bit more info than the heading explains) 🙂 Feel free to reword.
```suggestion

The following shows how to point to the root `node_modules` directory, rather than to the example's local one.

```